File: android/app/src/main/java/com/manuscripta/student/data/model/SessionStatus.java
package com.manuscripta.student.data.model;

/**
 * Enum representing the status of a learning session.
 */
public enum SessionStatus {
    /**
     * Session is currently active/ongoing.
     */
    ACTIVE,

    /**
     * Session has been paused.
     */
    PAUSED,

    /**
     * Session completed normally.
     */
    COMPLETED,

    /**
     * Session was cancelled before completion.
     */
    CANCELLED
}

File: windows/ManuscriptaTeacherApp/Main/Models/Enums/SessionStatus.cs
namespace Main.Models.Enums;

/// <summary>
/// Session status enum as defined in Validation Rules §2D(1)(c).
/// </summary>
public enum SessionStatus
{
    /// <summary>
    /// Session is currently active/ongoing.
    /// </summary>
    ACTIVE,

    /// <summary>
    /// Session has been paused.
    /// </summary>
    PAUSED,

    /// <summary>
    /// Session completed normally.
    /// </summary>
    COMPLETED,

    /// <summary>
    /// Session was cancelled before completion.
    /// </summary>
    CANCELLED
}
